x-pack/plugins/fleet/server/routes/setup/handlers.ts

```diff
@@ -76,12 +62,19 @@ export const createFleetSetupHandler: RequestHandler<
   }
 };
 
-export const FleetSetupHandler: RequestHandler = async (context, request, response) => {
-  const soClient = context.core.savedObjects.client;
-  const esClient = context.core.elasticsearch.client.asCurrentUser;
-
+// TODO should be removed as part https://github.com/elastic/kibana/issues/94303
+export const FleetAgentSetupHandler: RequestHandler<
```

Comment:
I know this will be removed but 😄  use `fleetAgentSetupHandler` (lower camelcase)